spec/erb_lint/cli_spec.rb

```diff
@@ -217,7 +223,7 @@ def run(_processed_source)
           context 'when no errors are found' do
             let(:args) { ['--enable-linter', 'linter_without_errors', linted_dir] }
 
-            it 'shows no that errors were found to stderr' do
+            it 'shows that no errors were found to stderr' do
```

Comment:
```suggestion
            it 'shows that no errors were found to stdin' do
```

Reply:
While we're at it!

Reply:
Was this a mistake @exterm ? You mean `stdout` right?